docs/protocol/network/messages/heartbeat.md

```diff
@@ -0,0 +1,29 @@
+# Heartbeat
+
+The hearbeat protocol has as main purpose to notify to other peers that the node is still active and running. This information may be relevant for managing a list of active peers.
+
+The heartbeat protocol is defined by using `ping` and `pong` messages and it allows to implement different strategies to react to peer inactivity.
```

Comment:
Usually, AFAIK it depends on the node implementation. Different implementations may define different strategies on how to avoid inactive peers. I suppose we will define such strategy in our implementation, and, perhaps, we will even support that the final user may override some parameters (e.g. maximum inactivity period of a connected node).